src/extension.ts

```diff
@@ -71,6 +81,11 @@ export function activate(context: ExtensionContext) {
 	context.subscriptions.push(disposable);
 }
 
+function getCamelSettings() {
+	const camelXMLConfig = workspace.getConfiguration('camel');
```

Comment:
maybe we should make 'camel' a constant?

Reply:
OMG, how does TypeScript is working???
by remplacing with constant, it is putting the constant name instead of the value...

it is sending this kind of data then:

```
Params: {
    "settings": {
        "SETTINGS_TOP_LEVEL_KEY_CAMEL": {
            "Camel catalog version": "2.22.1"
        }
    }
}
```

Reply:
Interesting

Reply:
so found out why. when we want to use a variable for a property key, we need to force the compute with [ ], see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names

Reply:
good to know. I haven't seen that before.